tests/test_core.py

```diff
@@ -1116,6 +1116,17 @@ def test_exceptions(self):
       self.set_setting('SUPPORT_LONGJMP', support_longjmp)
       self.do_run_from_file(test_file('core', 'test_exceptions.cpp'), test_file('core', 'test_exceptions_caught.out'))
 
+  # We have LTO tests covered in 'wasmltoN' targets, but they don't run as a
+  # part of Emscripten CI, so we make a separate LTO test here.
+  @with_both_exception_handling
+  def test_exceptions_lto(self):
+    self.set_setting('EXCEPTION_DEBUG')
+    self.emcc_args += ['-flto']
+    self.maybe_closure()
```

Comment:
Maybe move this to `test_other` and remove `maybe_closure` which is core specific?

Reply:
This uses `with_both_exception_handling` decorator. Is there a way to reuse it in test_other.py?

Reply:
We would move the `with_both_exception_handling` decorator to `runner.py` in that case.    Is is not just the native exceptions you are interested in testing here?

Reply:
I see... this is the first time a `-fwasm-exceptions` test has been added to test_other.py which is why its not setup to make that easy.  

I don't feel strongly but normally when we want to add specific tests that are covered by the "modes" in core we put them instead in "other"... otherwise things get confusing when you run `test_exceptions_lto` in `wasm2lto` mode..  I guess in this case it would just be redundant..

Reply:
I moved the test to test_other.py, with `with_both_exception_handling` and `SUPPORT_LONGJMP` options removed. You are right that what we try to test here is wasm EH + LTO so maybe we don't need to test the full combinations of options there.